block/block.go

```diff
@@ -33,6 +36,42 @@ type Body struct {
 	Txs tx.Transactions
 }
 
+// GetMockBlock creates and returns a mock block for testing purposes.
+func GetMockBlock(parentId thor.Bytes32) *Block {
+	tx1 := new(tx.Builder).Clause(tx.NewClause(&thor.Address{})).Clause(tx.NewClause(&thor.Address{})).Build()
+	tx2 := new(tx.Builder).Clause(tx.NewClause(nil)).Build()
+
+	privKey := string("dce1443bd2ef0c2631adc1c67e5c93f13dc23a41c18b536effbbdcbcdb96fb65")
```

Comment:
Use `genesis.DevAccounts()`

Reply:
Also here afaik `genesis.DevAccounts()` just returns a list of addresses but we need a mock block instead. What am I missing here?

Reply:
Yes, it's a list, you can pick any of them